### PR TITLE
docs: tweak some examples for consistency and correctness

### PR DESCRIPTION
**Pull request by knownasilya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,9 +47,9 @@ import Ember from 'ember';
 import { CanMixin } from 'ember-can';
 
 export default Ember.Route.extend(CanMixin, {
-  beforeModel: function() {
+  beforeModel() {
     if (!this.can('write post')) {
```
Comment (blimmer) [inline README.md]: Should this have a call to super as well?

Comment (knownasilya) [inline README.md]: Like in an else, probably..

Comment (knownasilya) [inline README.md]: Done

Comment (blimmer) [inline README.md]: I actually think super should always be called. For instance, what if some mixin model hook loads info about the user and their abilities (not a good pattern, just as an example).

I propose that this reads:

```js
beforeModel() {
  this._super(...arguments);

   if (!this.can('write post')) {
     return this.transitionTo('index');
  }
}
```

Comment (knownasilya) [inline README.md]: What if super depends on a value being returned? Do we assign the result to a var and return if can't write?

Comment (blimmer) [inline README.md]: good point, might make sense. the reason i bring all this up is that i often see issues where some mixin implements some global change, like page tracking or something, and that is not called when overriding these lifecycle events.

**Comment by knownasilya** [comment]
Done

**Comment by blimmer** [comment]
awesome - thank you!

**Pull request merged**